### PR TITLE
Add specs: Blast Sepolia

### PR DESCRIPTION
**Pull request by AsharibAli**
**Where are you in the process of adding the spec?**

I’ve completed drafting the spec, but I’m looking forward to adding more integrated functionalities to enhance its capabilities further.

**Why did you choose to add this spec to Lava?**

I’m planning to integrate this spec into my `create-blast-dapp` project, especially if I win in the ongoing competition. I aim for it to support the basic operational features of the Blast chain, ensuring smooth functionality.

**What help do you need?**

So far, the documentation has been incredibly clear and helpful, so I haven’t encountered any confusion. Everything seems straightforward.

**Provide some links to the Blast (Docs or whitepaper) for reference**

Blast Website: https://blast.io/
Documentation: [About Blast - Blast Developer Documentation](https://docs.blast.io/about-blast)

### CODE REVIEW

File: cookbook/specs/spec_add_blast.json [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+                  {
+                    "expected_value": "0xA30769"
+                  }
+                ]
+              },
+              {
+                "name": "pruning",
+                "values": [
+                  {
+                    "latest_distance": 1024
```
Comment (omerlavanet) [inline cookbook/specs/spec_add_blast.json:48]: this means any query 1025+ blocks in the past will either need to go to archive or fail, our usual rule of thumb is 24h of blocks so 24*3600/averageBlockTime(s)

**Review by omerlavanet — CHANGES_REQUESTED** [review]
please attach the result of scripts/test_spec_full.sh with the spec file to the PR

**Comment by AsharibAli** [comment]
@omerlavanet can you please look at it, brother!

**Comment by omerlavanet** [comment]
> please attach the result of scripts/test_spec_full.sh with the spec file to the PR

have you tested it?

File: cookbook/specs/spec_add_blast.json [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+                  }
+                ]
+              }
+            ],
+            "extensions": [
+              {
+                "name": "archive",
+                "cu_multiplier": 3,
+                "rule": {
+                  "block": 2048
```
Comment (omerlavanet) [inline cookbook/specs/spec_add_blast.json:62]: this is still directing traffic to archive pretty quickly,
it's usually pruning latest distance-3%

**Review by omerlavanet — DISMISSED** [review]
looks good, just missing the rule fix and attaching the script run result to showcase it worked

**Comment by AsharibAli** [comment]
> have you tested it?

Could you please test it for me, I'm having some problems in my dev environment!

**Comment by AsharibAli** [comment]
@omerlavanet please test the following, if your environment is ready for testing, I'm having some issues!

**Pull request merged**